docs/sources/tutorials/send-logs-to-loki.md

```diff
@@ -223,23 +315,38 @@ With this configuration, {{< param "PRODUCT_NAME" >}} connects directly to the L
    ```
 
    {{< /code >}}
+   <!-- INTERACTIVE ignore END -->
 
-1. Call the `/-/reload` endpoint to tell {{< param "PRODUCT_NAME" >}} to reload the configuration file without a system service restart.
+2. Call the `/-/reload` endpoint to tell {{< param "PRODUCT_NAME" >}} to reload the configuration file without a system service restart.
```

Comment:
```suggestion
1. Call the `/-/reload` endpoint to tell {{< param "PRODUCT_NAME" >}} to reload the configuration file without a system service restart.
```

Is it a requirement on the Killercoda side to have incrementing numbers in steps? Or just a result of edits?

For reference: https://grafana.com/docs/writers-toolkit/write/markdown-guide/#numbered-lists

Reply:
Hi @clayton-cornell not all and great catch! I believe  my vscode just auto increments and fail to catch them all